src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -52,14 +52,12 @@ object ZincRunnerWorkerConfig {
   }
 }
 
-// The list in this docstring gets clobbered by the formatter, unfortunately.
-//format: off
 /**
  * <strong>Caching</strong>
  *
  * Zinc has two caches:
  *   1. a ClassLoaderCache which is a soft reference cache for classloaders of Scala compilers.
- *   2. a CompilerCache which is a hard reference cache for (I think) Scala compiler instances.
+ *   1. a CompilerCache which is a hard reference cache for (I think) Scala compiler instances.
```

Comment:
I think this should be 2 instead of 1?

Reply:
It's supposed to be `1`. The reason is was formatting incorrectly before is because, confusingly, every item in a Scaladoc list should begin with `1. `:
https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html#markup